staticfiles/js/crowdsource.routes.js

```diff
@@ -134,9 +134,13 @@
       templateUrl: '/static/templates/contributors/arcolife.html'
     })
      
-	.when('/contributors/AditiNath', {
+    .when('/contributors/AditiNath', {
       templateUrl: '/static/templates/contributors/aditinath.html'
     })
+    
+    .when('/contributors/DeenAariff', {
+      templateUrl: '/static/templates/contributors/deenaariff.html'
```

Comment:
it doesn't look like you have a page built out yet. Could you please add a page with a short paragraph with your background.